src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -96,22 +96,22 @@
     property that we could hook but we can't in this case.
   -->
   <Target Name="DeployNetFrameworkBuildHost" BeforeTargets="_GetPackageFiles;AssignTargetPaths" Condition="'$(DesignTimeBuild)' != 'true'">
-    <!-- When we're not doing source build we package two variants: net6.0 and net472 for the broadest compability. -->
+    <!-- When we're not doing source build we package two variants: $(NetVS) and net472 for the broadest compability. -->
     <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
       <_NetFrameworkBuildHostProjectReference Include="..\..\..\Workspaces\Core\MSBuild.BuildHost\Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj">
         <TargetFramework>net472</TargetFramework>
         <ContentFolderName>BuildHost-net472</ContentFolderName>
       </_NetFrameworkBuildHostProjectReference>
       <_NetFrameworkBuildHostProjectReference Include="..\..\..\Workspaces\Core\MSBuild.BuildHost\Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj">
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>$(NetVS)</TargetFramework>
```

Comment:
Is `$(NetVS)` guaranteed to be in the set of `$(NetRoslynSourceBuild)`? And in this case we'd need the lowest version of all of them, or otherwise we could end up with say a 7.0 targeting app that consumes this as a 7.0 library, but the underlying build host is targeting 8.0 and things are broken if the user doesn't have 8.0 on their machine too.

(Longer term, this is probably a very special case where we could leave this as hard-coded since there's little code in this and it's almost entirely just shelling out to MSBuild, but there's a bit more work that needs to happen first.)